hashicorp/boundary-ui
# Chore concurrently use yarn

## Description

Address discrepancy with the usage of concurrently. Change using `npm` in favour of `yarn` within concurently scripts.

## How to Test

Run in each addon and UI individually the yarn scripts that have been changed. Succesfully run them and follow with the terminal log to make sure it call the related tasks concurrently. i.e: run `yarn lint` within `admin/ui` and that should trigger `lint:hbs`, `lint:js` and `lint:sass`.

## Checklist:
